test/functional/test_framework/test_framework.py

```diff
@@ -101,6 +101,7 @@ def __init__(self):
         self.bind_to_localhost_only = True
         self.set_test_params()
         self.parse_args()
+        self.rpc_timeout = int(self.rpc_timeout * self.options.factor) # optionally, increase timeout by a factor
```

Comment:
```suggestion
        self.rpc_timeout = self.rpc_timeout * self.options.factor # optionally, increase timeout by a factor
```

Is this still needed?

Reply:
I think it is. When I execute a test like this it throws the one error we talked about previously (*the range poll function*). 

```shell
./test/functional/p2p_tx_download.py                                                     (4s 89ms)  
2020-05-02T23:36:00.054000Z TestFramework (INFO): Initializing test directory /var/folders/7q/4ffytzk562dd2ky4bfg9_w7h0000gn/T/bitcoin_func_test_54bxj
085
2020-05-02T23:36:00.247000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/Users/brakmic/projects/bitcoin-dev/btc-factor/test/functional/test_framework/test_framework.py", line 452, in start_nodes
    node.wait_for_rpc_connection()
  File "/Users/brakmic/projects/bitcoin-dev/btc-factor/test/functional/test_framework/test_node.py", line 217, in wait_for_rpc_connection
    for _ in range(poll_per_s * self.rpc_timeout):
TypeError: 'float' object cannot be interpreted as an integer
```